driver/razermouse_driver.c

```diff
@@ -4885,7 +4890,11 @@ static struct razer_mouse_device *find_mouse(struct hid_device *hdev)
     if (!m_intf)
         return NULL;
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 4, 0)
+    dev = device_find_child(&m_intf->dev, &hid_bus_type, dev_is_on_bus);
+#else
     dev = device_find_child(&m_intf->dev, hid_bus_type, dev_is_on_bus);
+#endif
```

Comment:
Hm I guess before we were passing `struct bus_type **` to the function so `dev_is_on_bus` function actually got a `void *` to a pointer? I wonder how that was actually working, I don't see a deference of the `void *` in `dev_is_on_bus`